tests/unit/store/waveform.spec.js

```diff
@@ -6,12 +6,12 @@ import { ping_get_available_data } from "@/store/modules/waveform/actions";
 import { arry, new_arry } from "./../js_utils/waveform_data_provider.js";
 import { get_available_data_regex } from "@/store/modules/waveform/url_regex";
 import {
-  system_status_when_calibrating_regexp,
-  all_mantarray_commands_regexp,
+  //  system_status_when_calibrating_regexp,
```

Comment:
remove unneeded commented code

Reply:
it looks like this commented code is still present

Reply:
Sorry I removed other comments, now its resolved.